csrc/ops/normalization.cpp

```diff
@@ -249,14 +253,14 @@ auto norm_properties_from_num_dims(
   std::vector<bool> inner_broadcast_mask(kNumberOfDims, false);
 
   for (const auto idx : c10::irange(kOuterNumDims)) {
```

Comment:
Should we make `outer_reduction_axes` and `inner_reduction_axes` both use `std::vector<int64_t>`?